### PR TITLE
bugfix: change CRI version 

### PR DESCRIPTION
**Pull request by mond77**

### CODE REVIEW

**Comment by mond77** [comment]
```
root@lmaster:~/install/sealos# sealos run labring/kubernetes:v1.25.6 labring/calico:v3.24.1 --single
```
```
root@lmaster:~/install/sealos# sealos run labring/kubernetes:v1.26.0
```
```
root@lmaster:~/install/sealos/cmd/sealos# kubectl get nodes
NAME      STATUS   ROLES           AGE     VERSION
lmaster   Ready    control-plane   4m45s   v1.26.0
```

**Comment by cuisongliu** [comment]
add test using sealos-action ci ?

**Comment by cuisongliu** [comment]
> > add test using sealos-action ci ?
> 
> I cann't understand. And how to use it?

https://github.com/labring/sealos/blob/main/.github/workflows/test_run.yml

add test unit for upgrade ?

File: pkg/runtime/upgrade.go [hunk 1]
```diff
@@ -199,3 +207,28 @@ func (k *KubeadmRuntime) tryUncordonNode(ip, nodename string) error {
 	}
 	return nil
 }
+
+func (k *KubeadmRuntime) changeCRIVersion() error {
+	if err := modifyFile("/etc/image-cri-shim.yaml", "version: v1alpha2", "version: v1"); err != nil {
```
Comment (cuisongliu) [inline pkg/runtime/upgrade.go:212]: 1. Does this just seem to be a local modification? What if the node executed by sealos is not in the cluster?
2. Image-cri-shim is executed at each node, and the modification should modify all nodes.

**Review by cuisongliu — CHANGES_REQUESTED** [review]

File: pkg/runtime/upgrade.go [hunk 2]
```diff
@@ -44,11 +44,14 @@ const (
...
 	if versionutil.Compare(version, V1250) {
-		logger.Info("start change ClusterConfiguration up to v1.25")
+		logger.Info("Change ClusterConfiguration up to v1.25 if need.")
 		if err := k.ChangeConfigToV125(); err != nil {
 			return err
 		}
 	}
+	if versionutil.Compare(version, V1260) {
+		logger.Info("Kubernetes v1.26 will not support CRI v1alpha2. You will need to upgrade to containerd v1.6.0 or higher.")
+	}
```
[the remark this answers is not in the document]
Comment (mond77) [reply pkg/runtime/upgrade.go:59]: done

**Review by cuisongliu — APPROVED** [review]

**Pull request merged**